qsimcirq/qsim_simulator.py

```diff
@@ -126,8 +126,9 @@ class QSimOptions:
 
     Args:
         max_fused_gate_size: maximum number of qubits allowed per fused gate.
-            Depending on the capabilities of the device qsim runs on, this
-            usually has best performance when set to 3 or 4.
+            For circuits of less than 22 qubits, set this to 2 or 3 for peak
+            performance. Larger circuits (with >= 22 qubits) perform best
+            with this set to 3 or 4.
```

Comment:
This might be oversimplified. Perhaps add "usually" or "typically"?

Reply:
Done.